Big Support Update

EchoStretch opened the pull request:
Thanks To @buzzer-re , @BestPig , @idlesauce and @zecoxao

EchoStretch committed:
More Fw Support

EchoStretch committed:
Added Back

EchoStretch committed:
Create build.yml

EchoStretch committed:
typo

EchoStretch committed:
fix mov_cr3_rax_mov_ds

EchoStretch committed:
fix cr3_phys_addr

EchoStretch committed:
Added More Offsets

EchoStretch committed:
finished 5.xx

EchoStretch committed:
fix notify

EchoStretch committed:
required fix

EchoStretch committed:
small fix

EchoStretch committed:
Update main.c

EchoStretch committed:
Update main.c

EchoStretch committed:
finished 6.xx offsets

EchoStretch committed:
Finished 3.xx Offsets

EchoStretch committed:
Added More 4.xx Offsets

EchoStretch committed:
Update 7xx Offsets

EchoStretch committed:
4.50 Offsets Added

Thanks to @zecoxao

EchoStretch committed:
Update 4.00 offsets.c

Thanks to @zecoxao

EchoStretch committed:
typo

sleirsgoevy commented on lib/dl.c:
```diff
@@ -16,6 +16,6 @@ void* dlsym(void* handle, const char* name)
 {
     if(!p_sceKernelLoadStartModule)
-        p_sceKernelLoadStartModule = dlsym((void*)0x2001, "sceKernelLoadStartModule");
+        p_sceKernelLoadStartModule = dlsym((void*)0x1, "sceKernelLoadStartModule");
```
0x2001 is the correct value for the webkit process.

sleirsgoevy commented on lib/syscalls-ps5.py:
```diff
@@ -76,7 +76,7 @@ def get_syscalls():
 mov r10, [rel addr__dynlib_dlsym]
 push r10
-mov edi, 0x2001
+mov edi, 0x1
```
Same as above.

sleirsgoevy commented on lib/syscalls.py:
```diff
@@ -21,7 +21,7 @@
         print('push r9')
         print('push rax')
-        print('mov edi, 0x2001')
+        print('mov edi, 0x1')
```
Same as above.

EchoStretch committed:
Finished 4.02 Offsets

Thanks @zecoxao

EchoStretch committed:
Adding 8.00 Info (Not Working)

EchoStretch committed:
Adding More 8.00 Info

EchoStretch committed:
revert sdk

idlesauce committed:
DLC Support
For DLCs to work and games to not show up as trial, you have to install the pkgs with this version of kstuff loaded.
Based on https://github.com/PS5Dev/Byepervisor/blob/57204cbd7bd26ed4623634d52b0f60f40d630087/hen/src/fpkg.cpp#L82
Currently only has offsets for 4.03 and 5.50.

EchoStretch committed:
Merge pull request #1 from idlesauce/bd-jb

DLC Support

EchoStretch committed:
Added Other DLC OFFSETS 3.00-8.00

EchoStretch committed:
Update main.c

EchoStretch committed:
fixed 8.00 pfsclearkeys

EchoStretch committed:
Added 8.20 Support

EchoStretch committed:
Added 8.40/8.60 Parcelle Support

EchoStretch committed:
Added Full Support 8.40/8.60

Fixed Shellcore Patching 8.40+ 
Thanks @idlesauce

EchoStretch committed:
Added 9.00 Full Support

EchoStretch committed:
Added 9.xx Full Support

EchoStretch committed:
Added 10.0x Offsets ( Not Working )
